py-polars/polars/dataframe/frame.py

```diff
@@ -1609,6 +1611,203 @@ def raise_on_copy(msg: str) -> None:
 
         return out
 
+    @overload
+    def to_jax(
+        self,
+        return_type: Literal["array"] = ...,
+        *,
+        device: jax.Device | str | None = ...,
+        label: str | Expr | Sequence[str | Expr] | None = ...,
+        features: str | Expr | Sequence[str | Expr] | None = ...,
+        dtype: PolarsDataType | None = ...,
+        order: IndexOrder = ...,
+    ) -> jax.Array: ...
+
+    @overload
+    def to_jax(
+        self,
+        return_type: Literal["dict"],
+        *,
+        device: jax.Device | str | None = ...,
+        label: str | Expr | Sequence[str | Expr] | None = ...,
+        features: str | Expr | Sequence[str | Expr] | None = ...,
+        dtype: PolarsDataType | None = ...,
+        order: IndexOrder = ...,
+    ) -> dict[str, jax.Array]: ...
+
+    def to_jax(
```

Comment:
Can we mark this as unstable so that we can iterate over the API a little bit in the future?

We should do the same for `to_torch`.

Reply:
Oof... I was absolutely convinced I had already done so for `to_torch`, but apparently am confidently hallucinating like an LLM! Will update both accordingly ;)

Reply:
Done :ok